test/src/test/java/org/corfudb/AbstractCorfuTest.java

```diff
@@ -102,6 +104,12 @@ public void cleanupScheduledThreads() {
         scheduledThreads.clear();
     }
 
+    @After
+    public void cleanPerTestTempDir() {
+        deleteFolder(new File(PARAMETERS.TEST_TEMP_DIR),
+                false);
+    }
+
```

Comment:
this is now confusing:

1. getTempDir is NEVER used, so why are we keeping it?
2. alternatively, getTempDir/cleanTempDir/deleteTempDir should all operate on the same dir

Why not leave getTempDir as a wrapper for obtaining a temp dir, and simply modify it to use PARAMETERS? and then continue invoking getTempDir in all the tests? This way, it is easy to change one place  if needed, in the future.